website/src/pages/extensions/index.tsx

```
@@ -1,24 +1,744 @@
-    <iframe
-      title="Extensions Catalog"
-      src="https://registry.podman-desktop.io/"
-      className="w-full min-h-[2000px] h-full"
-    />
+    <div
+      className="fixed inset-0 flex items-center justify-center z-250 p-4"
+      style={{ backgroundColor: 'rgba(0, 0, 0, 0.85)' }}
+      onClick={onClose}
+      tabIndex={-1}
+      onKeyDown={handleKeyDown}
+      role="presentation">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)